src/main.ts

```diff
@@ -49,6 +51,7 @@ axios.interceptors.response.use(
 // Vue config
 Vue.config.productionTip = isDebugMode
 Vue.config.errorHandler = function(error, vm, info) {
+  console.error(error || info)
```

Comment:
The client-side error handling works well but we need the raw error logged to console.